rerun_py/src/catalog/connection_handle.rs

```diff
@@ -456,4 +460,96 @@ impl ConnectionHandle {
 
         Ok(stores)
     }
+
+    pub fn get_chunk_ids_for_dataframe_query(
```

Comment:
I'd rather have any new such helper delegate to `ConnectionClient`, but let's at least have a todo here.